tools/kangax/run-kangax.py

```diff
@@ -0,0 +1,142 @@
+# limitations under the License.
+
+import subprocess
+import sys
+import os
+
+from argparse import ArgumentParser
+
+PATH = os.path.dirname(os.path.abspath(__file__))
+
+TERM_RED = '\033[1;31m'
+TERM_GREEN = '\033[1;32m'
+TERM_YELLOW = '\033[1;33m'
+TERM_EMPTY = '\033[0m'
+
+
+def separator_line():
+    print("{0}=====================================\n {1}".format(TERM_YELLOW, TERM_EMPTY))
+
+
+def get_results(filename, no_of_lines=1):
+    res = []
+    file = open(filename,'r')
+    lines = file.readlines()
+    last_lines = lines[-no_of_lines:]
+    for line in last_lines:
+        res.append(line)
+    file.close()
+    return res
+
+
+def run_testsuite(suite,engine):
+    separator_line()
+    print("{0}Running kangax suite(s):\n {1}{2}".format(TERM_YELLOW, 'es6\n es2016plus\n' if suite=='""' else suite, TERM_EMPTY))
+    separator_line()
+
+    if engine != 'local':
+        engine = engine
+    else:
+        engine = '../../escargot'
+
+    os.chdir(PATH + '/../../test/kangax/')
+    if suite == '""':
+        es6res = open("./kangaxES6Res.txt", "w")
+        es6plusres = open("./kangaxES6PLUSRes.txt", "w")
+        nodeFails = open("./kangaxRunErrors.txt", "w")
+        subprocess.call(['node', './escargot.js', engine, 'es6'], stdout = es6res, stderr = nodeFails)
+        print("{0}ES6 RESULTS:\n{1}".format(TERM_YELLOW, TERM_EMPTY))
+        grepProc = subprocess.Popen(['grep -hr "actual: false" ./kangaxES6Res.txt'], stdout=subprocess.PIPE, shell=True)
+        (fails, err) = grepProc.communicate()
+        print(''.join(get_results("./kangaxES6Res.txt", 2)))
+        print("{0}ES6 Fails:\n{1}".format(TERM_RED, TERM_EMPTY))
+        print(''.join(fails))
+        print("{0}Errors:\n{1}".format(TERM_RED, TERM_EMPTY))
+        es6res.close()
+        nodeFails.close()
+        with open("./kangaxRunErrors.txt", "r") as lines:
+            print(lines.read())
+        print('\n')
+
+        nodeFails = open("./kangaxRunErrors.txt", "w")
+        subprocess.call(['node', './escargot.js', engine, 'es2016plus'], stdout = es6plusres, stderr = nodeFails)
+        print("{0}ES6PLUS RESULTS:\n{1}".format(TERM_YELLOW, TERM_EMPTY))
+        grepProc = subprocess.Popen(['grep -hr "actual: false" ./kangaxES6PLUSRes.txt'], stdout=subprocess.PIPE, shell=True)
+        (fails, err) = grepProc.communicate()
+        print(''.join(get_results("./kangaxES6PLUSRes.txt", 2)))
+        print("{0}ES6PLUS FAILS:\n{1}".format(TERM_RED, TERM_EMPTY))
+        print(''.join(fails))
+        print("{0}Errors:\n{1}".format(TERM_RED, TERM_EMPTY))
+        es6plusres.close()
+        nodeFails.close()
+        with open("./kangaxRunErrors.txt", "r") as lines:
+            print(lines.read())
+        print('\n')
+    else:
+        res = open("./kangaxRes.txt", "w")
+        nodeFails = open("./kangaxRunErrors.txt", "w")
+        subprocess.call(['node', './escargot.js', engine, suite], stdout = res)
```

Comment:
It seems that `nodeFailes` is omitted for stderr here.

Reply:
That's a good catch, i missed this one!